pmoves/tools/inject_github_pat_from_gh_cli.py

```
@@ -103,7 +144,45 @@ def verify_scope(scopes: set[str]) -> None:
+    """Return True if `token` can actually query the repo's actions/runners.
+
+    Repo-scoped behavioral check — NOT `/user` (which a gist-only token passes,
+    giving a false positive). The token is injected via an env-isolated
+    GH_TOKEN with any ambient GH_TOKEN/GITHUB_TOKEN stripped, so a poisoned
+    ambient env var can't mask a bad token under test (the exact failure mode
+    that caused the runner-bootstrap saga).
+    """
+    env = {k: v for k, v in os.environ.items() if k not in ("GH_TOKEN", "GITHUB_TOKEN")}
+    env["GH_TOKEN"] = token
+    try:
+        result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)